client/app/components/packages/landuse-form/zoning-text-amendment.hbs

```diff
@@ -0,0 +1,51 @@
+{{#let @form as |form|}}
```

Comment:
Oops, we also need to toggle this section based off of if there is a ZR action. I'll add

Reply:
thanks!